### PR TITLE
Add disable for level 99 XP Orbs

### PR DESCRIPTION
**Pull request by jamesahhh**
Figured this seemed like an easy first contribution. Adds a simple toggle to disable all 99 XP Orbs from showing up in the Render list for the Overlay.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java [hunk 1]
```diff
@@ -118,6 +118,10 @@ public Dimension render(Graphics2D graphics)
 
 			for (XpGlobe xpGlobe : xpChangedQueue)
 			{
+				if (config.disable99s() && xpGlobe.getCurrentLevel() == 99)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java]: Would it be better to instead prevent the globe from being added/updated in onExperienceChanged()?

File: runelite-client/src/test/java/net/runelite/client/plugins/xpglobes/XPGlobePluginTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package net.runelite.client.plugins.xpglobes;
```
Comment (Adam-) [inline runelite-client/src/test/java/net/runelite/client/plugins/xpglobes/XPGlobePluginTest.java]: Add license header to all new files.

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java [hunk 3]
```diff
@@ -137,20 +142,20 @@ private void renderProgressCircle(Graphics2D graphics, XpGlobe skillToDraw, int
 		graphics.setRenderingHint(RenderingHints.KEY_STROKE_CONTROL, RenderingHints.VALUE_STROKE_PURE);
 
 		drawProgressArc(
-			graphics,
-			x, y,
-			config.xpOrbSize(), config.xpOrbSize(),
-			PROGRESS_RADIUS_REMAINDER, radiusToGoalXp,
-			5,
-			config.progressOrbOutLineColor()
+				graphics,
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java]: Undo all this extra formatting

File: runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java (hunk 1 above)
Comment (jamesahhh) [reply runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java]: Will remove this, this was my initial way to get rid of it. I ended up moving it into the addXPGlobe() called in onExperienceChanged().

File: runelite-client/src/test/java/net/runelite/client/plugins/xpglobes/XPGlobePluginTest.java (hunk 2 above)
Comment (jamesahhh) [reply runelite-client/src/test/java/net/runelite/client/plugins/xpglobes/XPGlobePluginTest.java]: Header added, formatting removed

**Pull request closed without merging**